scripts/ci/validate-wallet-swagger.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+if [[ $TRAVIS_COMMIT == "" ]]; then
+    echo "TRAVIS_COMMIT is unspecified"
+    exit 1
+fi
+
+spec_path="https://raw.githubusercontent.com/input-output-hk/cardano-sl/$TRAVIS_COMMIT/wallet-new/spec/swagger.json"
+validate="curl http://online.swagger.io/validator/debug?url=$spec_path"
```

Comment:
@Martoon-00 Idea: As this is an external service, and as such it might not work 100% of the time, should be capture the possibility curl fails either with exit code `1` or with a non-200 error but still allow CI to be completed in green? I suspect other folks would be extremely angry with us if we break the entire CI 😁 

Another way to see this, is that we should either:

- Add another job like `hlint` in the build matrix that can still fail separately;
- Make this check step optional;

What do you think? (I might even prefer 1., but I would be curious to hear what you think)!

Reply:
@adinapoli-iohk Well, you can see the first approach implemented here
I'm afraid that in case of that swagger job failing CI fill be just green, and no one will notice that swagger is invalid unless he opens job matrix. If so, this validation looks pretty useless.
Do you know something about it? I've spent whole evening trying to get only swagger job failing to see how it would look like, but failed.

In this sense approach #0 (remain validate mandatory, fail only on 200 OK responses with error report) may be better, however I'm afraid of breaing CI if that swagger validator one day goes mad

Reply:
> I'm afraid that in case of that swagger job failing CI fill be just green, and no one will notice that swagger is invalid unless he opens job matrix. If so, this validation looks pretty useless.

I see what you mean, but I think this is for now the safest bet for not hurting other developers productivity. Let me see if I can kindly ask devops to review the work and merge it in a timely fashion 😉